news/2023/2023-08-28-medal-hunter-titles-second-coming.md

```diff
@@ -0,0 +1,70 @@
+
+The community was trying their best to *hunt* for the solution, but after an enormous number of attempts that they thought would surely work, insanity started creeping in, and hope was slowly starting to fade. *On this journey of basking in the shade of an ailing mind*, the community had received various hints. The first notable one was from Ephemeral mentioning that this medal could involve "ARG" (alternate reality game) which blasted medal hunters' sanity away even more. The possibilities seemed endless. Another hint was a video recorded by [Glassive](https://osu.ppy.sh/users/12453848), one of medal hunters at [cavoe's osu! event (COE)](/wiki/Community/cavoe's_osu!_event), featuring [peppy](https://osu.ppy.sh/users/2) himself!
+
+<div align="center">
+  <video width="50%" controls>
+    <source src="https://assets.ppy.sh/media/news/coe_video_with_peppy.mp4?20230902" type="video/mp4" preload="none">
+  </video>
+</div>
+
+peppy said the solution may need to be altered because it depends on something that could be changed in the future. The only two possibilities were lazer and site. For those that guessed site and thought difficulty colors, they used the hint to their fullest of potentials. There was a downside to this hint though. While peppy basically confirmed this theory about diff's colors being correct, medal hunters also got a little mislead.
+After peppy said anyone, including new players would be able to get it, they started to believe it can't be 4* difficulty or above that give the medal, which was completely wrong and resulted in community taking much more time to solve it. That subhint mentioning "Bask in the shade of an ailing mind", referring to basking in the **RGB** shades of an **Insane** diff in particular was completely ignored, because newcomers to osu! would not be able to get this medal.
```

Comment:
```suggestion
After peppy said anyone, including new players would be able to get it, they started to believe it can't be performed on a difficulty higher than 4 star, which was completely wrong and resulted in community taking much more time to solve it. That subhint mentioning "Bask in the shade of an ailing mind", referring to basking in the **RGB** shades of an **Insane** diff in particular was completely ignored, because newcomers to osu! would not be able to get this medal.
```